test/testBrowserLanguage.js

```diff
@@ -0,0 +1,139 @@
+import browserLanguage from '../src/nginxconfig/util/browser_language';
+
+class MockLocales{
+    static languages = [];
+
+    static language = null;
+
+    static IntlBackup = null;
+
+    static navigator = null;
+
+    static setNavigatorLanguages(langs){
+        MockLocales.languages = langs;
+        return this;
+    }
+
+    static setNavigatorLanguage(lang){
+        MockLocales.language = lang;
+        return this;
+    }
+
+    static setDateTimeLocale(locale) {
+        const newDateTimeForMat = new Intl.DateTimeFormat(locale);
+        MockLocales.IntlBackup = Intl;
+        if (!locale){
```

Comment:
```suggestion
        if (!locale) {
```
Formatting consistency